### ISSUE TITLE
[ISSUE] Billable Usage Download failing

### ISSUE THREAD

**Issue by amayhew-apixio** [databricks/databricks-sdk-py#308]
**Description**
0.6.0 Python SDK request for billable_usage.download() throws exception because API response is not json.

**Reproduction**
```
from databricks.sdk import AccountClient

a = AccountClient(
        host = "https://accounts.cloud.databricks.com",
        account_id = "XXX",
        client_id = "XXX",
        client_secret = "XXX"
        )

a.billable_usage.download(start_month="2023-07", end_month="2023-08")
```

**Expected behavior**
Would expect either the API to respond with json assignable to a variable.
i.e. json_results = a.billable_usage.download(start_month="2023-07", end_month="2023-08")

**Debug Logs**
This trace is being thrown from the exception handling on line 1009 in `databricks/sdk/core.py` since the response from the API endpoint is 'content-type': 'text/plain;charset=utf-8' and is a CSV "file"

```
Traceback (most recent call last):
  File "/Users/amayhew/Documents/work/code/apx-aws-costs/./databricks_costs.py", line 13, in <module>
    a.billable_usage.download(start_month="2023-07", end_month="2023-08")
  File "/Users/amayhew/Documents/work/code/venv/lib/python3.11/site-packages/databricks/sdk/service/billing.py", line 447, in download
    self._api.do('GET', f'/api/2.0/accounts/{self._api.account_id}/usage/download', query=query)
  File "/Users/amayhew/Documents/work/code/venv/lib/python3.11/site-packages/databricks/sdk/core.py", line 1014, in do
    raise self._make_nicer_error(message=message) from None
databricks.sdk.core.DatabricksError: workspaceId,timestamp,clusterId,clusterName,clusterNodeType,clusterOwnerUserId,clusterCustomTags,sku,dbus,machineHours
[response truncated]

```
Error message returned is the CSV "file" from the request.

**Other Information**
 - OS: [e.g. macOS]
 - Version: 0.6.0

**Additional context**

**Comment by mgyucht** [comment]
Thanks for reporting this! We're aware of the issue and are working on a fix in the coming weeks.

**Comment by mgyucht** [comment]
This has been fixed in the v0.11.0 release of the SDK.
